amanda-guo/accessibility-checker
# Adding data to charts

## Describe your changes

- Refactored the file into modules
- Added the ability to visualize the parsed JSON data into a bar chart and a pie chart
- Added the ability to take the URL from the input and call the API on that URL to display relevant charts

## Steps to test
- Run the app
- Use different URLs to check that the charts outputted are different
- Be careful: can easily run out of API credits on Wave

## Checklist before requesting a review
